docs/content/users/quickstart.md

```diff
@@ -1,274 +1,230 @@
+    New Craft CMS projects can be created from the official [starter project](https://github.com/craftcms/craft) using DDEV’s [`composer create` command](../users/usage/commands.md#composer):
 
-        # Run the Craft installer:
-        ddev craft install
-        ddev launch
-        ```
+    ```bash
+    # Create a project directory and move into it:
+    mkdir my-craft-project
+    cd my-craft-project
 
-        Third-party starter projects can by used the same way—just substitute the package name when running `ddev composer create`.
+    # Set up the DDEV environment:
+    ddev config --project-type=craftcms --docroot=web --create-docroot
 
-    === "Existing projects"
+    # Boot the project and install the starter project:
+    ddev start
+    ddev composer create -y --no-scripts craftcms/craft
 
-        You can start using DDEV with an existing project, too—just make sure you have a database backup handy!
+    # Run the Craft installer:
+    ddev craft install
+    ddev launch
+    ```
 
-        ```bash
-        # Clone an existing repository (or navigate to a local project directory):
-        git clone https://github.com/example/example-site my-craft-project
-        cd my-craft-project
+    Third-party starter projects can by used the same way—just substitute the package name when running `ddev composer create`.
 
-        # Set up the DDEV environment:
-        ddev config --project-type=craftcms
+=== "Existing projects"
 
-        # Boot the project and install Composer packages:
-        ddev start
-        ddev composer install
+    You can start using DDEV with an existing project, too—just make sure you have a database backup handy!
 
-        # Import a database backup and open the site in your browser:
-        ddev import-db --file=/path/to/db.sql.gz
-        ddev launch
-        ```
+    ```bash
+    # Clone an existing repository (or navigate to a local project directory):
+    git clone https://github.com/example/example-site my-craft-project
+    cd my-craft-project
+
+    # Set up the DDEV environment:
+    ddev config --project-type=craftcms
+
+    # Boot the project and install Composer packages:
+    ddev start
+    ddev composer install
+
+    # Import a database backup and open the site in your browser:
+    ddev import-db --file=/path/to/db.sql.gz
+    ddev launch
+    ```
 
     Craft CMS projects use PHP 8.1 and MySQL 8.0, by default. You can override these settings during setup with [`config` command flags](./usage/commands.md#config) or after setup via the [configuration files](./configuration/config.md).
 
     !!!tip "Upgrading or using a generic project type?"
         If you previously set up DDEV in a Craft project using the generic `php` project type, update the `type:` setting in `.ddev/config.yaml` to `craftcms`, then run [`ddev restart`](../users/usage/commands.md#restart) apply the changes.
 
-    ### Running Craft in a Sub-directory
+### Running Craft in a Sub-directory
```

Comment:
Un-hyphenate _sub-directory_ to be more consistent with DDEV and Craft’s docs and [common usage](https://www.merriam-webster.com/dictionary/subdirectory):

```suggestion
### Running Craft in a Subdirectory
```

Reply:
and one comment about the rest of the changes. i haven't touched any of the quickstarts text for this PR. wanted to tackle that in the next steps. but all the suggested changes are definitely improvements.